docs/walkthrough/loading-models.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 1
+---
+
+# Loading Models
+
+<!-- THE MARKDOWN (.md) FILE IS GENERATED FROM THE NOTEBOOK (.ipynb) FILE -->
+
+Substratus makes it very easy to import your existing ML models into Substratus.
```

Comment:
Replace `Substratus makes it very easy` with something more descriptive and less sales'y sounding like: `Substratus can import existing ML models with a single API call`.

Reply:
Simple, easy, obvious, straightforward - all terms we should work to avoid even if we think the system is well described by them. To an outside user, it wont be so straightforward.

Reply:
done